src/pybind/mgr/cephadm/module.py

```diff
@@ -887,16 +887,16 @@ def _executable_path(self, conn, executable):
         return executable_path
 
     def _run_cephadm(self,
-                     host: str,
-                     entity: Optional[str],
-                     command: str,
-                     args: List[str],
-                     addr: Optional[str] = None,
-                     stdin: Optional[str] = None,
-                     no_fsid=False,
-                     error_ok=False,
-                     image: Optional[str] = None,
-                     env_vars: Optional[List[str]] = None,
+                     host,                # type str
+                     entity,              # type str
+                     command,             # type str
+                     args,                # type List[str]
+                     addr = "",           # type Optional[str]
+                     stdin = "",          # type Optional[str]
+                     no_fsid = False,     # type Optional[bool]
+                     error_ok = False,    # type Optional[bool]
+                     image = False,       # type Optional[str]
+                     env_vars= None       # type Optional[List[str]]
```

Comment:
why change this to the old style type annotation?

Reply:
Because I didn't know what is the preferred annotation type... by your comment I assume it is the "new" annotation style. I will change.